### PR TITLE
lots of indenting

### PR DESCRIPTION
**Pull request by sskhandek**

### CODE REVIEW

File: src/modules/registration/components/input/input.component.js [hunk 1]
```diff
@@ -60,50 +60,47 @@ const formatOption = (str) => {
...
+		if (val == 'graduationYear') {
 			var reg = new RegExp("^[12][0-9]{3}$");
 			//console.log(reg.test(this.props.store.userData[val]))
     		return reg.test(this.props.store.userData[val]);
 		}
-		if(val == 'age') {
+		if (val == 'age') {
 			var reg = new RegExp(/^\d{1,2}$/);
 			//console.log(reg.test(this.props.store.userData[val]))
     		return reg.test(this.props.store.userData[val]);
```
Comment (redsn0w422) [inline src/modules/registration/components/input/input.component.js:81]: not indented

Comment (redsn0w422) [reply src/modules/registration/components/input/input.component.js:81]: I'll fix it

Comment (redsn0w422) [reply src/modules/registration/components/input/input.component.js:81]: Oh weird it's okay in Sublime but not on GitHub

Comment (sskhandek) [reply src/modules/registration/components/input/input.component.js:81]: wut

**Review by redsn0w422 — APPROVED** [review]

**Pull request merged**